platform/common/core/generic/vault/helpers_test.go

```diff
@@ -106,6 +117,111 @@ func (p *populator) Populate(rws *vault.ReadWriteSet, rwsetBytes []byte, namespa
 	return nil
 }
 
+type marshaller struct{}
```

Comment:
We can reuse the marshaller that is already written

Reply:
but it is under the fabric package. What I can do is to write a dummy marshaller.

Reply:
hmm i think it would make more sense either to write another marshaler or to move this test in the fabric package and test only the fabric implementation. but if you think it's a lot of work this is also fine, but please document it, because now it seems that it is testing a specific implementation of the marshaler.

Reply:
so, now we test against a local marshaller and a fabric-based marshaller.